### PR TITLE
Need to test if page changes if different profile is seen

### PR DESCRIPTION
**Pull request by J0Santos**
Avatar, nick, won_games and lost_games are now getting pulled from db. If the User seeing the profile is not the same as the profile itself, they wont see the settings wheel.

TODO: get info for game history. Need help with this, understanding what variable I have access to. Can I get avatar from opponent, for example? By checking function all_history (game_history.service.ts) it looks like I cant.

### CODE REVIEW

File: .env.example
Comment (ncameiri42) [inline .env.example:1]: Este file não pode ser deleted

File: frontend/src/views/ProfileView.vue [hunk 1]
```diff
@@ -1,22 +1,73 @@
...
+userData.nick = userNick;
+
+const isOwnProfile = computed(() => {
+	return userProfile.value.nick === userData.nick;
+});
+
+const fetchUserProfile = async () => {
+	try {
+		let url = process.env.VUE_APP_BACKEND_URL + '/users/getUsers';
+		const response = await fetch(url);
```
Comment (ncameiri42) [inline frontend/src/views/ProfileView.vue]: Deve ser enviado o Bearer, uma vez que todos os endpoints serão protegidos

**Comment by Dmarceli** [comment]
Why pr if it need testing??

File: frontend/src/views/ProfileView.vue (hunk 1 above)
Comment (J0Santos) [reply frontend/src/views/ProfileView.vue]: nao percebo, tas a dizer isto?

`let url = process.env.VUE_APP_BACKEND_URL + '/users/getUsers/' + userData.nick;`

Comment (J0Santos) [reply frontend/src/views/ProfileView.vue]: ou isto?

`const response = await fetch(url,
      {
        headers: {
          'Authorization': `Bearer ${token}`
        }
      });`

**Comment by J0Santos** [comment]
Foi mal, ja testei, ta a funcionar @Dmarceli. Mas agora deixa-me entao resolver os problemas que o nuno disse

File: frontend/src/views/ProfileView.vue (hunk 1 above)
Comment (ncameiri42) [reply frontend/src/views/ProfileView.vue]: const response = await fetch(url, { headers: { 'Authorization': Bearer ${token} } });

File: frontend/src/router/index.ts [hunk 2]
```diff
@@ -30,9 +30,14 @@ const router = createRouter({
     },
     {
       path: '/profile',
-      name: 'profile',
+      name: 'myProfile',
       component: ProfileView
-    }
+    },
+		{
+			path: '/profile/:nick',
```
Comment (Dmarceli) [inline frontend/src/router/index.ts:37]: Not sure if it’s using nick property in user but it should user intra_nick, nick can be changed

**Review by Dmarceli — APPROVED** [review]
Lgtm

**Pull request merged**